tests.py

```diff
@@ -170,7 +170,7 @@ def test_advanced_add(self):
         with open(self.config_file) as f:
             # check that property is really flushed out to the config?
             content = f.read().encode('ascii')
-            self.assertIn(b'identityfile "/tmp/idfilecheck.rsa"', content)
+            self.assertIn(b'IdentityFile "/tmp/idfilecheck.rsa"', content)
             self.assertIn(b"stricthostkeychecking yes", content)
```

Comment:
Should these (StrictHostKeyChecking and UserKnownHostsFile not also assert CamelCase?